docker/ssl/ca/example.cossacklabs.com.crt

```diff
@@ -1,20 +1,22 @@
 -----BEGIN CERTIFICATE-----
-MIIDSzCCAjOgAwIBAgIUWM4d35tmkpYprCzSPW9Zp20W6nswDQYJKoZIhvcNAQEL
-BQAwIjEgMB4GA1UEAwwXZXhhbXBsZS5jb3NzYWNrbGFicy5jb20wIBcNMjEwMTEx
-MTUwOTE4WhgPMjA3MDEyMzAxNTA5MThaMCIxIDAeBgNVBAMMF2V4YW1wbGUuY29z
-c2Fja2xhYnMuY29tMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAuv8/
-gKeEuvrsz9YvAnEAkuWAphs2NIDVoTg/wjDha53Z5E1Sf2G7ZwRHlyLBRLlGq2UD
-5zgYHdDUWLW2oKsMg2nXD9w5HQmzgWAYd/jMR1VJdGHbJOSg5BbGEzIK/2M4pUk9
-k/8WRzqzDp4NtkV0+2xIdmlYSZCwLhjEeyovuTikD03nPf9XGA1bQ6ibB3JBqDI/
-fC4AYglGZg+jOD8v/b6fNJrEvo6BaVt8mcJhqUi2sZ6mk3K8woLNQcFw2UZY9LO2
-Y4eOChtjNxly5HMZPEIsWl+UqM2z+3BuSITuOWwjBB9bXzRR7VgZWALQXFN2g8Ns
-SW1mE/X7Nf8mnKFiQQIDAQABo3cwdTAPBgNVHRMBAf8EBTADAQH/MCIGA1UdEQQb
-MBmCF2V4YW1wbGUuY29zc2Fja2xhYnMuY29tMB0GA1UdDgQWBBStPuUpucne4KD4
-ymBXi1Vxsd+VvzAfBgNVHSMEGDAWgBStPuUpucne4KD4ymBXi1Vxsd+VvzANBgkq
-hkiG9w0BAQsFAAOCAQEAckX0jB/BWtzFdwOV22lnY7kqoj1sFFhMDZrx5NOFXBMo
-oigM3mP6ICdGBmWpf6/9jwFKLhPS3JmKF2K6aaWcMDK4alvHO4aJSLfw9rmASOZ2
-ngmPlHkiLMWTXjQnohsGXkq3d9h9wRiy49z2SLROqd3eDPui/DgdMC7r5x5MCfQD
-/b21l8ldWc5XwZ306o0juYmd4wTj8AdQeJ7tQxcj3IXuIucNhgG5bZxQzuId/YwZ
-AHjrc8/A8NTpo0TcZ6c2bv+040MYPdnoczqeLX6Pwvle7o+ZjLaRw21Y8/6Xy0Ik
-+z/rZZ4TH8/Auk5KjcF27k9kVD4akgzQmkqRoU80eQ==
+MIIDpTCCAo2gAwIBAgIUS1qfqghbdhstE+EJe5KBklJGWQIwDQYJKoZIhvcNAQEL
```

Comment:
can you remind how you generate this certs?

Reply:
Just using the generation script that we have in tests folder